File: minions/12.Agua en botella/Agua_en_botella.c
#include <stdio.h>
#include <stdlib.h>

int main(){
    int T;
    scanf("%d", &T);

    for (int j = 0; j < T; j++){
        int n, c;
        scanf("%d %d", &n, &c);

        int C_uso = c;
        int baldes = 1;

        int Cn[n];
        for (int i = 0; i < n; i++){
            scanf("%d", &Cn[i]);
        }

        for (int i = 0; i < n; i++){
            C_uso -= Cn[i];
            if (C_uso <= 0){
                baldes++;
                C_uso = c;
            }   
        }
        
        printf("%d\n", baldes);
    }

    return EXIT_SUCCESS;
}

File: minions/12.Agua en botella/Agua_en_botella.py

T = int (input())

for j in range (T):
    n,c = [int(i) for i in input().split(" ")]
    C_uso = c
    baldes = 1
    
    Cn = [int(i) for i in input().split(" ")]
    for i in Cn:
        C_uso -= i
        if C_uso <= 0:
            baldes += 1
            C_uso = c
            
    print(baldes)